src/remote_adapter/remote_adapter_client/sdl_remote_adapter_client.h

```diff
@@ -0,0 +1,177 @@
+#pragma once
+
+#include <string>
+#include <utility>
+
+#include "rpc/client.h"
+#include "rpc/rpc_error.h"
+
+namespace lua_lib {
+
+class SDLRemoteTestAdapterClient {
+ public:
+  SDLRemoteTestAdapterClient(const std::string& host, uint32_t port);
+
+  /**
+   * @brief connected checks if client is connected to server
+   * @return true if connected otherwise false
+   */
+  bool connected() const;
+
+  /**
+   * @brief Sends open websocket request to server
+   * @param address - host name which should be opened by server
+   * @param port - number port which should be opened by server
+   * @return 0 in successful case, 1 - if client is not connected,
+   * 2 - in case of exception
+   */
+  int tcp_open(const std::string& address,uint32_t port);
```

Comment:
@SNiukalov There is mismatch function name and function brief. 

The same issue occurs in all `tcp_*` functions.

Reply:
@aderiabin Fixed in 1826a1e